packages/icon-builder/package.json

```diff
@@ -16,27 +21,30 @@
     "React",
     "react-component"
   ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/callemall/material-ui-icons.git"
-  },
-  "author": "Hai Nguyen",
-  "license": "MIT",
-  "dependencies": {
-    "yargs": "^4.3.2"
+  "scripts": {
+    "prebuild": "rimraf build",
+    "build": "npm run build:icons && npm run build:copy-files",
+    "build:icons": "node ./build.js --output-dir ./build --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --renameFilter ./filters/rename/material-design-icons.js",
+    "build:copy-files": "babel-node ./copy-files.js",
+    "test": "grunt"
   },
-  "peerDependencies": {},
```

Comment:
WE do: https://github.com/mbrookes/material-ui/blob/ad45aa88cfab602da1135a5b9f2923916248ba63/packages/icon-builder/copy-files.js#L64